src/kps/Main.java

```diff
@@ -23,6 +24,9 @@ public static void main(String[] args) throws XMLException, FileNotFoundExceptio
         Simulation simulation;
         try {
             simulation = SimulationXML.readSimulationFromFile(new FileInputStream(args[0]));
+            for (Mail m : simulation.getMail()){
+                m.setCalculatedRoute(simulation.buildCalculatedRoute(m.getFrom(), m.getTo(), m.getPriority()));
+            }
```

Comment:
Ok then you need to use this code to update the saved log file and then remove it.